Refactor _get_source_transforms to remove args

jackzhxng opened the pull request:
Refactor `_get_source_transforsm` to remove args

iseeyuan commented on examples/models/llama/export_llama_lib.py:
```diff
@@ -1196,12 +1259,25 @@ def _get_source_transforms(  # noqa
         this wil be a no-op.
         """
-        modelname = f"{modelname}_e"
+        # Create a mock args object with the necessary attributes
```
Is it to limit the extend of changes? It's better than Namespace, but I'm wondering if eventually we need to group args, and there can still be passing the entire args even not all fields are needed.

jackzhxng replied:
Yup this is just to limit the changes, after I do a final refactor of the quant source transform functions we can remove this

iseeyuan reviewed (approved):
It's a great improvement from passing Namespace directly. Thanks @jackzhxng !

jackzhxng committed:
Refactor _get_source_transforms to remove args (#10519)

Summary:
Refactor `_get_source_transforsm` to remove args


Reviewed By: iseeyuan

Differential Revision: D73800023

Pulled By: jackzhxng
